projects/1.16/assets/aiot-botania/aiotbotania/lang/zh_cn.json

```
@@ -1,51 +1,68 @@
-  "botania.page.manaGear11": "由于安装了$(2)植物魔法全能工具模组$(0)，新增了两个物品<br>$(1)锄$(0)和原版锄类似，但像其他工具一样使用$(4)魔力$(0)。<br>$(1)全能工具$(0)融合了$(1)魔钢剑$(0), $(1)镐$(0), $(1)斧$(0), $(1)锹$(0)和$(1)锄$(0)，并结合了它们的全部特性。",
-  "botania.page.livingrockTools7": "$(1)活石全能工具$(0)",
-  "botania.page.livingrockTools6": "$(1)活石锄$(0)",
-  "botania.page.livingwoodTools4": "$(1)活木锹$(0)",
-  "botania.page.livingwoodTools3": "$(1)活木镐$(0)",
-  "botania.page.livingwoodTools2": "$(1)活木斧$(0)",
-  "botania.page.livingwoodTools1": "$(1)活木剑$(0)",
-  "botania.page.livingwoodTools0": "植物魔法全能工具模组可以将$(1)活木$(0)制成工具。<br>这些工具有与$(1)木质$(0)类似的特性，却在可操作性，耐用性和一些调整方面有些优越。每当$(1)活木$(0)物品受到损坏时，如果物品栏中有$(4)魔力$(0)（例如$(4)魔力石板$(0)）可用，它将代替它。<br>除此之外，它们与原版工具一样。",
-  "item.aiotbotania.elementium_aiot": "源质钢全能工具",
-  "item.aiotbotania.elementium_hoe": "源质钢锄",
-  "item.aiotbotania.manasteel_hoe": "魔钢锄",
-  "botania.page.livingwoodTools5": "$(1)活木剪刀$(0)",
-  "botania.page.livingrockTools5": "$(1)活石剪刀$(0)",
-  "botania.page.livingrockTools4": "$(1)活石锹$(0)",
-  "botania.page.livingrockTools3": "$(1)活石镐$(0)",
-  "botania.page.livingrockTools2": "$(1)活石斧$(0)",
-  "botania.page.livingrockTools1": "$(1)活石剑$(0)",
-  "botania.page.livingrockTools0": "植物魔法全能工具模组可以将$(1)活石$(0)制成工具。<br>这些工具有与$(1)石质$(0)类似的特性，却在可操作性，耐用性和一些调整方面有些优越。每当$(1)活石$(0)物品受到损坏时，如果物品栏中有$(4)魔力$(0)（例如$(4)魔力石板$(0)）可用，它将代替它。<br>除此之外，它们与原版工具一样。",
-  "botania.page.livingwoodTools7": "$(1)活木全能工具$(0)",
-  "botania.page.livingwoodTools6": "$(1)活木锄$(0)",
-  "botania.entry.livingrockTools": "活石工具",
-  "botania.entry.livingwoodTools": "活木工具",
-  "item.aiotbotania.manasteel_aiot": "魔钢全能工具",
-  "item.aiotbotania.livingrock_aiot": "活石全能工具",
-  "item.aiotbotania.livingrock_hoe": "活石锄",
-  "item.aiotbotania.livingrock_shovel": "活石锹",
-  "item.aiotbotania.livingrock_pickaxe": "活石镐",
-  "item.aiotbotania.livingrock_axe": "活石斧",
-  "item.aiotbotania.livingrock_sword": "活石剑",
-  "item.aiotbotania.livingrock_shears": "活石剪刀",
-  "item.aiotbotania.livingwood_aiot": "活木全能工具",
-  "item.aiotbotania.livingwood_hoe": "活木锄",
-  "item.aiotbotania.livingwood_shovel": "活木锹",
-  "item.aiotbotania.livingwood_pickaxe": "活木镐",
-  "item.aiotbotania.livingwood_axe": "活木斧",
-  "item.aiotbotania.livingwood_sword": "活木剑",
-  "item.aiotbotania.livingwood_shears": "活木剪刀",
-  "aiotbotania.utilityMode": "§e放置火把和树苗",
-  "aiotbotania.hoeModePath": "§3耕作泥土和创建草径",
-  "aiotbotania.hoeMode": "§3耕作泥土",
-  "aiotbotania.toggleMode": "§6模式：",
-  "itemGroup.aiotbotania": "植物魔法全能工具"
+{
+  
+  "itemGroup.aiotbotania": "植魔全能工具",
+  "aiotbotania.toggleMode": "\u00a76模式:",
+  "aiotbotania.hoeModePath": "\u00a7a开垦土地与草径",
+  "aiotbotania.utilityMode": "\u00a7e放置火把与树苗",
+  "aiotbotania.mythicbotany.disabled": "请安装 MythicBotany！",
+  "aiotbotania.hoeMode": "\u00A73锄地",
+
+  "item.aiotbotania.livingwood_shears": "活木剪刀",
+  "item.aiotbotania.livingwood_sword": "活木剑",
+  "item.aiotbotania.livingwood_axe": "活木斧",
+  "item.aiotbotania.livingwood_pickaxe": "活木镐",
+  "item.aiotbotania.livingwood_shovel": "活木铲",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "item.aiotbotania.livingwood_shovel": "活木锹",
```
原版译名。